examples/cookbook/src/lib.rs

```diff
@@ -218,7 +218,7 @@ mod tests {
         let contract_instance = MyContract::new(contract_id, wallet_1.clone());
 
         let call_handler = contract_instance.methods().initialize_counter(42);
-        let mut executable_call = call_handler.get_executable_call().await?;
+        let mut executable_call = call_handler.get_executable_call().await?.prepare().await?;
```

Comment:
I don't like the UX of this, it is yet another chain of `method()` to just do a contract call.
From what I understand, the suggestion was to let the user do her thing, and then only do one pass for requesting resources (both for fees and resources) from the node.